kube/kd/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
```

Comment:
Because this is a task branch this change won't be respected, you'll have to just enter the credentials before executing any commands.